backend/inspirehep/disambiguation/cli.py

```diff
@@ -114,30 +116,22 @@ def _send_celery_group_disambiguation_task(uuids, batch_size):
 @disambiguation.command(name="not-disambiguated")
 @with_appcontext
 @click.option(
-    "-gs",
-    "--group-size",
-    type=int,
-    default=100,
-    help="Batch size for celery group primitive",
+    "--celery-batch-size", type=int, default=5, help="Batch size for celery task chunks"
 )
 @click.option(
-    "-bs", "--batch-size", type=int, default=5, help="Batch size for celery task chunks"
+    "-bs",
+    "--batch-size",
+    type=int,
+    help="Number of records to disambiguate, if not passed all records with at least one not disambiguated will be sent to the queue",
```

Comment:
--total-records maybe? it's a bit confusing for me

Reply:
yes, it's better 👍